src/HonzaBotner/appsettings.CvutFit.json

```diff
@@ -52,6 +52,10 @@
             "648686713845055489": 2
         }
     },
+    "ReminderOptions": {
+        "CancelEmojiName": "\uD83D\uDD07",
+        "JoinEmojiName": "✋"
+    },
```

Comment:
NIT: I would rather use 🗑️ and 🔔/⏰.

Reply:
:+1:  Updated in https://github.com/fit-ctu-discord/honza-botner/pull/156/commits/88f9cbb734e5361487ea96451d05da4565ed67c0